### DISCUSSION TITLE
[Feature] Drop files link for people without an account

### DISCUSSION THREAD

**Discussion by Y0ngg4n** [immich-app/immich#2656] [Feature Request]
### The feature

Hey would  be nice to have some kind of upload feature for people who are not registered. Something like a "drop to album" feature.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
When you create a shared album link, you can enable uploading.

**Reply by Y0ngg4n** [reply]
yes have seen that it is already implemented :)
